src/core/map.h

```diff
@@ -1469,7 +1471,7 @@ protected slots:
 private:
 	typedef std::vector<MapColor*> ColorVector;
 	typedef std::vector<Symbol*> SymbolVector;
-	typedef std::vector<Template*> TemplateVector;
+	typedef std::vector<std::unique_ptr<Template>> TemplateVector;
```

Comment:
The use of `unique_ptr` obsoletes `delete placeholder;` in `Importer::validate()` (file_import_export.cpp). In fact, the `delete` leads to a crash.

Reply:
Okay. Will look into this tonight.

Reply:
Release notes updated.
Side note: I meant to test the binaries once more before publishing the draft release, but accidenty hit the wrong button.